### ISSUE TITLE
Mastodon likes of a known contact's post not federated to my instance

### ISSUE THREAD

**Issue by wiwie** [friendica/friendica#8529]
When comparing a post of one of my contacts on his own instance:
https://libranet.de/display/0b6b25a8-195e-9bf8-3022-872711162930
and my instance:
https://friendica.cwiwie.org/display/0b6b25a8-195e-9bf8-3022-872711162930
It seems that likes from Mastodon contacts are not federated to me.

Is this a bug or a feature?

**Comment by annando** [comment]
I have the feeling as if likes are only transmitted directly to the original author. I observed it many times - not only with likes on Friendica posts but also with likes on Mastodon posts.

I need to dig deeper into it, but my current guess is that it is a feature from Mastodon.

**Comment by wiwie** [comment]
Okay. Would it not be the task of the like receiving friendica instance to forward the likes to the other ones that liked the post on friendica? Is it the one that makes the like that has to send it to everyone?

**Comment by annando** [comment]
This is something that I have to have a look on.

**Comment by wiwie** [comment]
Is there any way I could provide more insight to help debugging this?

**Comment by annando** [comment]
No. I have got enough test accounts on several systems for this.

**Comment by annando** [comment]
Likes from Mastodon are transmitted as private posts, so they won't be transmitted to other systems.
